packages/cli/test/commands/items/list.test.ts

```diff
@@ -27,12 +27,56 @@ describe('items:list', () => {
       const expected = readFileSync(outputFixture('list-items.output.yaml'), 'utf-8');
       expect(ctx.stdout).to.contain(expected);
     });
+
+  customTest
+    .stub(ItemService.prototype, 'list', listFilterByClassificationNodeName as any)
+    .stdout()
+    .run([
+      'items:list',
+      '--classificationNodeName',
+      'pets',
+      ...testUserAuth,
+      ...testEnvironmentFile,
+    ])
+    .it('list items that the user has', ctx => {
+      const expected = readFileSync(
+        outputFixture('list-items.filter.by.classification.name.output.yaml'),
+        'utf-8'
+      );
+      expect(ctx.stdout.trim()).to.contain(expected.trim());
+    });
+
+  customTest
+    .stub(ItemService.prototype, 'list', listFilterByClassificationNodeNames as any)
```

Comment:
You can reduce the test maintenance burden by just checking the args sent to the stub. We really only need to test the argument parsing and error logic for the CLI right? (Example in tests for connections:accept)

Reply:
Also verifying the output, no?  Like that the YAML is in the right format.

Reply:
Like to be explicit, the acvtual command calls `service.list`/`service.listAll` and also `printYaml({ kind: 'Items' })` so both would be tested?

Reply:
For sure, but we should test that just where the output changes based on the different params, not just have a similar formatted file for each permutation of input args.

Reply:
hmm, well we also need to check the final YAML response output, as it will be different from the actual response from SDK.
what do you think?

Reply:
I agree with @joshbax189, if the output is the same then it's already tested, so no need to test it again.